app/scripts/app.constants.js

```diff
@@ -0,0 +1,66 @@
+/**
+* (c) 2016 Tieto Finland Oy
+* Licensed under the MIT license.
+*/
+'use strict';
+angular.module('dashboard')
+.constant(
+    'DEVICE', {
+        MOBILE: 1,
+        DESKTOP: 2
+    }
+)
+.constant(
+    'MENU', {
+        CLOSED: 0,
+        HALF: 1,
+        FULL: 2
+    }
+)
+.constant(
+    'BLOCKMODE', {
+        BOTH : 0,
+        UPPER : 1,
+        LOWER : 2
+    }
+)
+.constant(
+    'HOMEMODE', {
+        ALL : 0,
+        MEETINGS : 1,
+        ESIGN : 2
+    }
+)
+.constant(
+    'APPSTATE', {
+        APP: "app",
+        LOGIN: "app.login",
+        INFO: "app.info",
+        HOME: "app.home",
+        OVERVIEW: "app.overview",
+        MEETING: "app.meeting",
+        SIGNITEM: "app.signitem"
+    }
+)
+.constant(
+    'ROLE', {
```

Comment:
I recommend either using namespacing ('MTG.ROLE.XYZ') or prefixing ('MTGROLE.XYZ') to avoid confusion with other non-meeting features.